src/foam/u2/view/ScrollTableView.js

```diff
@@ -193,9 +202,28 @@
     function init() {
       this.onDetach(this.data$proxy.listen(this.FnSink.create({ fn: this.updateCount })));
       this.updateCount();
+
+      if ( this.memento )
+        this.currentMemento$ = this.memento.tail$;
+      else
+        this.currentMemento$ = this.memento$;
     },
 
     function initE() {
+      if ( this.currentMemento ) {
+        var id = this.currentMemento.head;
+        if ( ! isNaN(id) ) {
+          id = parseInt(this.currentMemento.head);
+        }
+
+        this.stack.push({
+          class: 'foam.comics.v2.DAOSummaryView',
+          data: null,
+          config: this.config,
+          id: id
```

Comment:
What does this line do?

Reply:
it pushes DAOSummaryView if current memento is not null
for example for url `http://localhost:8080/#admin.data:countryDAO:AT`
we will note display table we push DAOSummaryView for Australia